defaults/main.yml

```diff
@@ -21,3 +21,4 @@ ler53_account_key_size: 2048
 ler53_account_key_dir: /etc/ssl/lets_encrypt
 ler53_account_key_file_name: lets_encrypt_account.key
 ler53_new_cert_when_csr_changes: false
+ler53_letsencrypt_enviornment: https://acme-v01.api.letsencrypt.org/directory
```

Comment:
Can you remove this variable after you address the comments about using `default(omit)` below? That way if for whatever reason this URL changes in the upstream Ansible module, we don't have to update this variable and cut a new release of the role.